packages/url_launcher/url_launcher_web/example/integration_test/link_widget_test.dart

```diff
@@ -199,6 +391,27 @@ html.Element _findSingleAnchor() {
   return foundAnchors.single;
 }
 
+void _simulateClick(html.Element target) {
+  final html.MouseEvent clickEvent = html.MouseEvent('click');
+  clickEvent.initMouseEvent('click', /*bubbles*/ true);
```

Comment:
`initMouseEvent` seems to be [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/initMouseEvent), can we set `bubbles: true` into the [options](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/MouseEvent#options) of the constructor of `MouseEvent`?